### PR TITLE
SY-General Pset updates

### PR DESCRIPTION
**Pull request by larswik**

### CODE REVIEW

**Comment by larswik** [comment]
@SergejMuhic : Updates of existing Psets. Maybe you take a look before merge?

**Comment by SergejMuhic** [comment]
Bucket A, right?

File: IFC4x3/Properties/r/RuntimeAtFullLoad_1Wa6XnYx56p8rACT$SifuH/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RuntimeAtFullLoad_1Wa6XnYx56p8rACT$SifuH" Name="RuntimeAtFullLoad" UniqueId="60906871-8bb1-46cc-8d4a-31dfdcb29e11" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RuntimeAtFullLoad_1Wa6XnYx56p8rACT$SifuH/DocProperty.xml]: Missing data type. `IfcDuration` okay?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml [hunk 2]
```diff
@@ -19,6 +19,7 @@
 		<DocProperty xsi:nil="true" href="Power_13VYb5tj10cOr5hZsTbt3R" />
 		<DocProperty xsi:nil="true" href="NominalPowerConsumption_0PrrQ0qSaHuO00025QrE$V" />
 		<DocProperty xsi:nil="true" href="NumberOfPowerSupplyPorts_2zCP6ydR933h5ygkKmdVFt" />
+		<DocProperty xsi:nil="true" href="FireRating_0eTJm0qT0HuO00025QrE$V" />
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml:22]: Would have never have guessed that electrical devices also have a fire rating!

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ProtectiveDeviceTypeEarthingSwitch/DocPropertySet.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ProtectiveDeviceTypeEarthingSwitch" UniqueId="f3c8bce8-1e29-4630-832d-62ba8eed79c4" ApplicableType="IfcProtectiveDevice/(.EARTHINGSWITCH.)">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ProtectiveDeviceTypeEarthingSwitch/DocPropertySet.xml]: You apparently did not get the update for this. Sorry about that.
```suggestion
<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ProtectiveDeviceTypeEarthingSwitch" UniqueId="f3c8bce8-1e29-4630-832d-62ba8eed79c4" ApplicableType="IfcProtectiveDevice/EARTHINGSWITCH">
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Otherwise, looks good.

**Comment by larswik** [comment]
> Bucket A, right?

Think so yes.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml:22]: It was really important (ask Florent).

File: IFC4x3/Properties/r/RuntimeAtFullLoad_1Wa6XnYx56p8rACT$SifuH/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/r/RuntimeAtFullLoad_1Wa6XnYx56p8rACT$SifuH/DocProperty.xml]: I will add.

Comment (larswik) [reply IFC4x3/Properties/r/RuntimeAtFullLoad_1Wa6XnYx56p8rACT$SifuH/DocProperty.xml]: Done!

**Comment by larswik** [comment]
@SergejMuhic : Ok to merge now?

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**